packages/module/src/ChatbotConversationHistoryNav/ChatbotConversationHistoryNav.tsx

```diff
@@ -152,6 +156,7 @@ export const ChatbotConversationHistoryNav: FunctionComponent<ChatbotConversatio
   emptyState,
   noResultsState,
   isCompact,
+  title,
```

Comment:
This is a nit, but we can define the placeholder text here instead and then remove it from line 273.:

```suggestion
  title = 'Chat History',
```